<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyPageClass</class>
 <widget class="QWidget" name="DailyPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DailyPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxSearch">
     <property name="title">
      <string>搜索</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>日期介于</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEditStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>和</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEditEnd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>  金额介于</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dblSpinBoxStart">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>和</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dblSpinBoxEnd">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>  类别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CategoryComboBox" name="comboCategory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>  用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboUser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>  备注</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNote"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="MyTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/edit_add.png</normaloff>:/Images/edit_add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/edit_remove.png</normaloff>:/Images/edit_remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/Notepad2.png</normaloff>:/Images/Notepad2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCategory">
       <property name="text">
        <string>分类</string>
       </property>
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/Options.png</normaloff>:/Images/Options.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAuto">
       <property name="text">
        <string>自动</string>
       </property>
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/add_user.png</normaloff>:/Images/add_user.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearch">
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/find.png</normaloff>:/Images/find.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>82</width>
         <height>142</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>MyTableView.h</header>
  </customwidget>
  <customwidget>
   <class>CategoryComboBox</class>
   <extends>QComboBox</extends>
   <header>CategoryComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections/>
</ui>
